docs/en/sql-reference/sql-statements/Data Definition/CREATE RESOURCE.md

```diff
@@ -0,0 +1,132 @@
+---
+{
+    "title": "CREATE RESOURCE",
+    "language": "en"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# CREATE RESOURCE
+
+## Description
+
+    This statement is used to create a resource. Only the root or admin user can create resources. Currently supports Spark, ODBC, S3 external resources.
+    In the future, other external resources may be added to Doris for use, such as Spark/GPU for query, HDFS/S3 for external storage, MapReduce for ETL, etc.
+
+    Syntax:
+         CREATE [EXTERNAL] RESOURCE "resource_name"
```

Comment:
What's the meaning of EXTERNAL during create resource clause?

Reply:
EXTERNAL is an external resource, independent of Doris.

Reply:
Could you please provide a demo?